core/src/main/kotlin/agent/Dto.kt

```diff
@@ -30,6 +30,15 @@ data class AgentInfoDto(
     val agentType: String
 )
 
+@Serializable
+data class BasicAgentInfoDto(
```

Comment:
1. BasicAgentInfoDto duplicates fields from AgentInfoDto, fix it, for example, by creating in AgentInfoDto a new field with type 'BasicAgentInfoDto'
1. The name 'Basic' is not well for this. I think in a basic agent, we should have 'id', 'serviceGroup', etc instead of random fields.